Seneca-CDOT/telescope
# Issue 2963: Adding Next-i18next for front-end translations

## Issue This PR Addresses
fixes #2963


## Type of Change




## Description
I added next-i18next to be able to translate our front end web page. 

As of now, when you go to google and navigate to settings > language and set to spanish or english you should see the strings for the corresponding language on the page navigation. 




## Steps to test the PR
1.  MUST do `npm ci`
2.  `cp config/env.staging .env`
3.  `npm run dev`
4. Notice the top right navigation "about us" and "sign in" in english.
5. Go to chrome settings > language > set to display in spanish. This will reload your browser (dont freak out!)
6. refresh the web page, and you should see the strings for about us and sign in in spanish 
![chrome_OfA1aKljrh](https://user-images.githubusercontent.com/77639637/155828848-4487613c-1eaa-4dba-aebf-de75b3e9b6e9.jpg)
7. To go back to english, search in the settings for idioma (language in spanish). 



## Checklist
